src/js/tech/html5.js

```diff
@@ -1750,7 +1750,21 @@ Html5.resetMediaElement = function(el) {
    *
    * @see [Spec] {@link https://www.w3.org/TR/html5/embedded-content-0.html#dom-video-videowidth}
    */
-  'videoHeight'
+  'videoHeight',
+
+  /**
+   * Set the value of `crossorigin` from the media element. `crossorigin` indicates
+   * to the browser that should sent the cookies along with the requests for the
+   * different assets/playlists
+   *
+   * @method Html5#setCrossorigin
+   * @param {string} crossorigin
+   *         - anonymous indicates that the media should not sent cookies.
+   *         - use-credentials indicates that the media should sent cookies along the requests.
+   *
+   * @see [Spec]{@link https://html.spec.whatwg.org/#attr-media-crossorigin}
+   */
+  'crossorigin'
```

Comment:
can `crossorigin` be added to the list on line 1508? It's helpful for grepping the source.